src/components/SelectAvatar/index.js

```diff
@@ -37,14 +39,27 @@ const SelectAvatar = ({ setAvatarData }) => {
 
   return (
     <>
-      <Image source={avatarSource || defaultProfileImage} style={styles.image} />
-      <Button handleOnPress={onSelect} title={selectAvatar} titleColor={BLACK} />
+      <View style={styles.container}>
+        <Image source={profileCircles} />
+        {avatarUrl && !avatarSource ? (
+          <Image source={{ uri: avatarUrl }} style={styles.profileImage} />
+        ) : (
+          <Image source={avatarSource || defaultProfileImage} style={styles.profileImage} />
+        )}
+      </View>
+      <Button
```

Comment:
I think the style for the text is missing here, right?

Reply:
It is using the Button's style for the text. Button is a custom component, not the native one. Perhaps I could have called it 'CustomButton'.